editor/import/audio_stream_import_settings.cpp

```diff
@@ -471,7 +471,7 @@ void AudioStreamImportSettingsDialog::_settings_changed() {
 
 	updating_settings = true;
 	stream->call("set_loop", loop->is_pressed());
-	stream->call("set_loop_offset", loop_offset->get_value());
+	stream->call("set_loop_offset", double(loop_offset->get_value()));
```

Comment:
`Range::get_value()` already returns a `double`, though, so this cast doesn't do anything, does it?

Reply:
I think so too, but I adjusted it because of what I said in https://github.com/godotengine/godot/pull/87860#issuecomment-1923494238.  
I don't trust floating point values in computer systems, so I copied what someone more experienced did.

Reply:
But… yeah. I think this one should be safe. Though the one below is reading from VARIANT::FLOAT afiak (I know that it's a double under the hood… but there should still be in implicit conversion going on that we can make an explicit cast)